sdk/keyvault/keyvault-admin/CHANGELOG.md

```diff
@@ -1,15 +1,25 @@
 # Release History
 
-## 4.5.0-beta.2 (Unreleased)
+## 4.5.0 (Unreleased)
 
 ### Features Added
 
+Since 4.4.0:
+- Managed Identity can now be used in place of a SAS token to access the blob storage resource when performing backup and restore operations.
+
 ### Breaking Changes
 
+Since 4.5.0-beta.1:
```

Comment:
Was wondering how folks feel about including such entries. I feel like they add noise when reading a changelog and understanding what's breaking but I don't have a strong opinion. But if we did that (i.e. removed the delineation between breaking changes among betas and changes among releases) we can also avoid the `Since 4.4.0` callout above

Reply:
That's fair, I do tend to lean on the side of verbosity here since otherwise it means beta customers need to read between the lines a bit to figure out whether moving from beta to GA will cause any issues. But maybe if the number of people actually using the beta is small that isn't such a huge deal.

Reply:
We started doing this - at least for Key Vault - a while back. I don't think it ever got wide adoption, but we've been doing it for the past couple GAs for KV.

Reply:
The goal was that customers shouldn't have to read through all the betas just to get a sense for what was changed, especially if we added something in beta 1 and than removed it in beta 2. Don't make them human diff'ers. :smile:

Reply:
Disclaimer: continuing the conversation for fun and profit but this is in no way blocking / related to this PR anymore 😄 

Definitely agree, let's not make human differs out of customers! I think where my mind is at is:

1. Between beta.1 and beta.2 the changelog _should_ list the changes for that reason
2. Meaningful changes have one month in a beta
3. In a perfect world, going from beta.2 to GA is _only_ updating the service version, pakcage version, and swagger SHA so there should be no changes between beta.2 and GA
4. Then, GA to GA changes can be listed / summarized in a single entry so I can see what changed between the last and current GA version

Of course we don't live in a perfect world and there's probably archboard changes and such that we don't have space to put out in a beta for a release cycle. In that case yeah it makes sense to add what you have here and I think I will start doing that as well, but will aim to minimize beta.x to GA changes. Does that seem reasonable? Thinking through how I would create and read CHANGELOG entries...